packages/docusaurus/lib/client/serverEntry.js

```diff
@@ -13,7 +13,7 @@ import {Helmet} from 'react-helmet';
 import {getBundles} from 'react-loadable-ssr-addon';
 import Loadable from 'react-loadable';
 
-import manifest from '@build/assets-manifest.json'; //eslint-disable-line
+import manifest from '@generated/assets-manifest.json'; //eslint-disable-line
```

Comment:
I like this change. If we could remove `build/server.bundle.js` that'll be good too.